packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -46,62 +47,49 @@ class CreateCommand extends FlutterCommand {
 
   @override
   Future<int> runCommand() async {
-    if (argResults.rest.isEmpty) {
-      printError('No option specified for the output directory.');
-      printError(usage);
-      return 2;
-    }
+    if (argResults.rest.isEmpty)
+      throw new ToolExit('No option specified for the output directory.\n$usage', exitCode: 2);
 
     if (argResults.rest.length > 1) {
-      printError('Multiple output directories specified.');
+      String message = 'Multiple output directories specified.';
       for (String arg in argResults.rest) {
         if (arg.startsWith('-')) {
-          printError('Try moving $arg to be immediately following $name');
+          message += '\nTry moving $arg to be immediately following $name';
           break;
         }
       }
-      return 2;
+      throw new ToolExit(message, exitCode: 2);
     }
 
-    if (Cache.flutterRoot == null) {
-      printError('Neither the --flutter-root command line flag nor the FLUTTER_ROOT environment\n'
-        'variable was specified. Unable to find package:flutter.');
-      return 2;
-    }
+    if (Cache.flutterRoot == null)
+      throw new ToolExit('Neither the --flutter-root command line flag nor the FLUTTER_ROOT environment\n'
```

Comment:
nit: here and elsewhere, braces around if statement bodies that have more than one line.